### PR TITLE
Source snapshot: Mon Jul 20 16:22:04 PDT 2020

### PR DESCRIPTION
**Pull request by lamberta**
Projects and last commit:

- keras-team/keras-io: https://github.com/keras-team/keras-io/commit/f8d5db905cada5accd383b2410cffae381c17c00
- tensorflow/addons: https://github.com/tensorflow/addons/commit/11511e4303baa084894b1cda64ac5b68b4238e3d
- tensorflow/agents: https://github.com/tensorflow/agents/commit/dc7417b577c72af3026a761e01363e1587b256a1
- tensorflow/datasets: https://github.com/tensorflow/datasets/commit/2b0cf6b78b5e4d8f220f404899cb97ba53af5063
- tensorflow/docs: https://github.com/tensorflow/docs/commit/bc9207a8f07e9c3da4e64c0203d684ef3dfa3b96
- tensorflow/federated: https://github.com/tensorflow/federated/commit/e90385b14d20278119b0b678e94a64d5311c4e33
- tensorflow/graphics: https://github.com/tensorflow/graphics/commit/3dd63357f54f42ad60c6dc0daeb3e1739f7863be
- tensorflow/hub: https://github.com/tensorflow/hub/commit/ae8b68e853a54ba06bc1559cc57bb721b933465f
- tensorflow/hub: https://github.com/tensorflow/hub/commit/ae8b68e853a54ba06bc1559cc57bb721b933465f
- tensorflow/io: https://github.com/tensorflow/io/commit/34a45c4419547a0d0db861246c94eca6e11b0db0
- tensorflow/lattice: https://github.com/tensorflow/lattice/commit/2f2cf33156636d3194eab2475585f58968276dbd
- tensorflow/model-optimization: https://github.com/tensorflow/model-optimization/commit/15e730ab4788f9775c77e3576078d844bc4afcc3
- tensorflow/neural-structured-learning: https://github.com/tensorflow/neural-structured-learning/commit/6396bc56d3fabdf179082bf64797ce31e4bc23df
- tensorflow/probability: https://github.com/tensorflow/probability/commit/c01e62badfafbfb7ead3338d69d48ca01577a0d7
- tensorflow/quantum: https://github.com/tensorflow/quantum/commit/4739a390cd51d9d09bf3d376ff58f67b5991aeda
- tensorflow/swift: https://github.com/tensorflow/swift/commit/399644b8958f1794d8004342f415a266238605d8
- tensorflow/tensorboard: https://github.com/tensorflow/tensorboard/commit/03b7edf797bca656308322e7d628edf6a2cc966c
- tensorflow/tensorflow: https://github.com/tensorflow/tensorflow/commit/f5b9c2225584c79539ff6746b3417e8505443a4b
- tensorflow/tensorflow: https://github.com/tensorflow/tensorflow/commit/f5b9c2225584c79539ff6746b3417e8505443a4b
- tensorflow/tensorflow: https://github.com/tensorflow/tensorflow/commit/f5b9c2225584c79539ff6746b3417e8505443a4b
- tensorflow/tfjs-website: https://github.com/tensorflow/tfjs-website/commit/0ae0906fd1410b8cec91be448f1aa205be3a3e95
- tensorflow/tfx: https://github.com/tensorflow/tfx/commit/de0e6c7316039ee5f3b2ab3e7814da2da223b8bd

### CODE REVIEW

File: site/en-snapshot/tutorials/quickstart/advanced.ipynb [hunk 1]
```diff
@@ -12,7 +12,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 0,
+      "execution_count": null,
```
Comment (lamberta) [inline site/en-snapshot/tutorials/quickstart/advanced.ipynb:15]: Should normalize this in nbfmt, if not already
